[DOCS] Adds examples to the PUT dfa and the evaluate dfa APIs (#46966)
elastic/elasticsearch: docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc

// TESTRESPONSE[s/1567168659127/$body.$_path/]
// TESTRESPONSE[s/"version": "8.0.0"/"version": $body.version/]

The following example creates a job and specifies a training percent:

[source,console]
--------------------------------------------------
PUT _ml/data_frame/analytics/student_performance_mathematics_0.3
{
 "source": {
   "index": "student_performance_mathematics"
 },
 "dest": {
   "index":"student_performance_mathematics_reg"
 },
 "analysis":
   {
     "regression": {
       "dependent_variable": "G3",
       "training_percent": 70  <1>
     }
   }
}
--------------------------------------------------
// TEST[skip:TBD]

> <1> The `training_percent` defines how many percent of the data set will be used 

Reviewer: ```suggestion
<1> The `training_percent` defines the percentage of the data set that will be used 
```
